build.override.targets

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="WcfSetup" Condition="'$(WithCategories)' == 'OuterLoop'" BeforeTargets="Test">
+    <Exec Command="$(MySetupScript) > $(MyLogFile)" />
```

Comment:
Where are MySetupScript, MyLogFile and MyCleanupScript initialized?  We should probably choose better names (if we own the names), like WcfSetupScript.  We should probably also add to the Condition that these properties are not empty.